### ISSUE TITLE
Question on checkpoint

### ISSUE THREAD

**Issue by NiranjanRavi1993** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#74]
Hi,

      Thank you for a great repository. I have a couple of questions. I am trying to run the train.py from scratch. At the end of the training, I get a  checkpoint file. When I try to run the detect.py, I could see the checkpoint belongs to the last epoch? Is it a good approach to save the checkpoint file after each epoch? 
 
Also, while training I could see the loss for each epoch. But I would like to visualize the accuracy as well? An accuracy function is defined in utils.py! Can I use it?

I am trying to visualize the results which are happening to get a better understanding. 
Any guidance/inputs will be helpful. 

Thank you
